tests/integration/delete.bats

```diff
@@ -13,23 +13,27 @@ function teardown() {
 
 @test "runc delete" {
   # run busybox detached
-  runc run -d --console-socket $CONSOLE_SOCKET test_busybox
+  runc run -d --console-socket $CONSOLE_SOCKET testbusyboxdelete
   [ "$status" -eq 0 ]
 
   # check state
-  testcontainer test_busybox running
+  testcontainer testbusyboxdelete running
 
-  runc kill test_busybox KILL
+  runc kill testbusyboxdelete KILL
   [ "$status" -eq 0 ]
   # wait for busybox to be in the destroyed state
-  retry 10 1 eval "__runc state test_busybox | grep -q 'stopped'"
+  retry 10 1 eval "__runc state testbusyboxdelete | grep -q 'stopped'"
 
-  # delete test_busybox
-  runc delete test_busybox
+  # delete the container
+  runc delete testbusyboxdelete
   [ "$status" -eq 0 ]
 
   runc state test_busybox
   [ "$status" -ne 0 ]
+
+  if test -n "$(find /sys/fs/cgroup -wholename '*testbusyboxdelete*')"; then
```

Comment:
Does busybox support `-wholename`? This flag doesn't shown up in `busybox find --help`

Reply:
it seems to be supported:

```
$ podman run --rm -ti alpine find /usr -wholename '*cat*'
/usr/bin/bzcat
/usr/bin/fallocate
/usr/bin/lzcat
/usr/bin/lzopcat
/usr/bin/truncate
/usr/bin/xzcat
```